lib/select.js

```diff
@@ -365,7 +367,7 @@ var classBase = React.createClass({
           return React.createElement(
             'option',
             { key: index, value: child.props.value },
-            child.props.children
+            child.props.value
```

Comment:
This was generated when the build step ran, I did not explicitly make this change myself.